MAESTRO/utils/scATAC_HTMLReport_batch.py

```diff
@@ -0,0 +1,119 @@
+import os,sys
```

Comment:
this one should be inside MAESTRO/ folder as well?

Reply:
just to be consistent with the current folder structure although it makes no difference by using:
 "python " + SCRIPT_PATH +  "/utils/scATAC_HTMLReport_batch.py
or 
"python " + SCRIPT_PATH +  "scATAC_HTMLReport_batch.py"

inside the Snakefile.

the MAESTRO/utils folder mainly contains binary from UCSC utils.